Sources/SwiftDocC/Infrastructure/Link Resolution/PathHierarchy+Find.swift

```diff
@@ -101,7 +101,12 @@ extension PathHierarchy {
                 }
             }
             let topLevelNames = Set(modules.keys + [articlesContainer.name, tutorialContainer.name])
-            throw Error.notFound(remaining: Array(remaining), availableChildren: topLevelNames)
+            
+            if isAbsolute {
+                throw Error.moduleNotFound(remaining: Array(remaining), availableChildren: Set(modules.keys))
```

Comment:
Should this be gated behind a feature flag?

Reply:
It could. I just thought that the more specific error message was a nice safe change.

Reply:
Oh that's fine then – I thought it might be a breaking change.